lib/tictactoe.ex

```diff
@@ -1,32 +1,22 @@
 defmodule TicTacToe do
-  
-  def make_move() do
-    Board.create(3)
-  end
-
-  def make_move(selected_tile, game_status) do
-    modified_game_status = Board.place_marker(selected_tile,game_status)
-    change_player(modified_game_status, elem(modified_game_status,0), elem(modified_game_status,2))
-  end
-
-  def change_player(game_status, :o, :continue) do 
-    {:x, elem(game_status, 1), :continue}
-  end
-
-  def change_player(game_status, :x, :continue) do 
-    {:o ,elem(game_status, 1), :continue}
-  end
 
-  def change_player(game_status, player, :win) do
-    {player,elem(game_status, 1), :win }
+  def first_game_status do
+    {:x, Board.create, :first_turn}
   end
 
-  def change_player(game_status, player, :tie) do
-    {player, elem(game_status, 1), :tie}
-  end
+  def make_move(selected_tile, {player_symbol, board, _}) do
+    tile_taken = Board.check_tile_not_taken(board, selected_tile)
 
-  def change_player(game_status, player, :tile_already_selected) do
-    {player, elem(game_status, 1), :tile_already_selected}
+    if tile_taken == :tile_already_selected do
+      next_game_status({player_symbol, board, :tile_already_selected})
+    else
+      new_board = Board.place_marker(selected_tile, board, player_symbol)
+      turn_type = WinOrTieChecker.check_if_win_or_tie(new_board)
```

Comment:
As the module is called `WinOrTieChecker`, you could actually get away with calling the method something like `evaluate`. As it stands the method `check_if_win_or_tie` is quite long, and where booleans are returned it is more idomatic to name them using the question mark, eg: `win_or_tie?`.